src/main/groovy/life/qbic/samplestatus/reporter/services/RealLimsQueryService.groovy

```diff
@@ -38,6 +38,10 @@ class RealLimsQueryService implements LimsQueryService {
 
   private final String sampleStatusProperty
 
+  private SampleStatusMapper statusMapper
+
+  private List<String> ignoredStatusValues = ["Sequencing completed"]
```

Comment:
Instead of having it fixed, you can just make this a configurable property and pass its value as a constructor parameter. 

The advantage is that you are able to configure the ignored values by reading them from a property file or command line argument, no need to re-compile and re-deploy the application.